core/commands/dht.go

```diff
@@ -278,71 +229,51 @@ var provideRefDhtCmd = &cmds.Command{
 		cmdkit.BoolOption("verbose", dhtVerboseOptionName, "Print extra information."),
 		cmdkit.BoolOption(recursiveOptionName, "r", "Recursively provide entire graph."),
 	},
-	Run: func(req cmds.Request, res cmds.Response) {
-		n, err := req.InvocContext().GetNode()
+	Run: func(req *cmds.Request, res cmds.ResponseEmitter, env cmds.Environment) error {
+		nd, err := cmdenv.GetNode(env)
```

Comment:
Why the rename to `nd`?

Reply:
@magik6k I just wanna write it in the same way in all `core/commands/*.go`

Reply:
Yeah, makes sense. (I thought that `n` was more common/conststently used, but checking now it's not and `nd` is used about as often, I have no preference, so this LGTM)